CONTRIBUTING.md

```diff
@@ -90,8 +90,6 @@ git push -f
 ```
 
 ### Merging pull requests
-Pull requests must be merged by a core maintainer using the `Merge pull request` option.
```

Comment:
Please retain this section, and provide additional instructions to support auto-merge via the bot.